ush/ocnice_extractvars.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+source "${USHgfs}/extractvars_tools.sh"
+
+outdirpre=${1}
+varlist=${2}
+dataformat=${3}
+datares=${4}
+datacompress=${5}
+fhout_ocnice=${6}
+comout_rfcst_prod_ocnice=${7}
+cd "${outdirpre}" || true
```

Comment:
If `outdirpre` doesn't exist, wouldn't this cause issues later on when attempting to write the `outfile` to `outdirpre`?  Perhaps it would be better to either issue an error or create the directory if it doesn't exist.  Also, it may not be necessary to `cd` to it.

```suggestion
[[ -d "${outdirpre}" ]] || mkdir -p "${outdirpre}"
```

Reply:
You are correct. It would be better to create the directory if it doesn't exist to ensure no issues occur when writing `outfile` to `outdirpre`. After checking through the scripts, every variable defined has an absolute directory path, so a `cd` is not necessary and removing it will not impact anything. Adding these changes in.

Reply:
These changes have been added.